### ISSUE TITLE
`make update` fails to find `../../nixpkgs`

### ISSUE THREAD

**Issue by Jayman2000** [abathur/resholve#99]
When I run `make update`, I get an error about not being able to find a file:

```none
FileNotFoundError: [Errno 2] No such file or directory: '../../nixpkgs/pkgs/tools/misc/dgoss/default.nix'
make: *** [Makefile:53: docs/README.nixpkgs.md] Error 1
```

[(Full log)](https://gist.github.com/Jayman2000/7f99841e52cef03d7d2727d81e69500f)

I tried putting a copy of [Nixpkgs](https://github.com/NixOS/nixpkgs/) at `../../nixpkgs/`, but that didn’t fix the problem.

**Comment by abathur** [comment]
Egh. Yes. I should probably go back and remove the note suggesting `make update` for now :)

If you actually need this, nixpkgs needs to be a peer directory to `resholve/`. 

(This is all just single-sourcing examples embedded in the nixpkgs README to ensure they stay up-to-date with how resholve is actually used in nixpkgs. I'll reapproach this in a more self-contained way when I flake it.)

**Comment by abathur** [comment]
This should be ~fixed in 5b7ae0c. `make update` should now create a symlink inside of the directory that points to the source of the `nixpkgs` flake input.

You should be able to toss that nixpkgs clone out, if you haven't already.
